srcpkgs/matterbridge/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'matterbridge'
+pkgname=matterbridge
+version=1.26.0
+revision=1
+build_style=go
+go_import_path="github.com/42wim/matterbridge"
+conf_files="/etc/matterbridge.toml"
+short_desc="Bridge between various chatting services"
+maintainer="Luca Matei Pintilie <luca@lucamatei.com>"
+license="Apache-2.0"
+homepage="https://github.com/42wim/matterbridge/"
+changelog="https://github.com/42wim/matterbridge/releases/"
+distfiles="https://github.com/42wim/matterbridge/archive/refs/tags/v$version.tar.gz"
+checksum=00e1bbfe3b32f2feccf9a7f13a6f12b1ce28a5eb04cc7b922b344e3493497425
+
+post_install() {
+	vconf matterbridge.toml.sample matterbridge.toml
```

Comment:
this is really more of an example config, maybe it would be better to `vsconf` all 3 examples and the user needs to write their own conf based on them

Reply:
there's only 2 config files in the newest release, so to save the next person the trouble I made it into a for loop